static/app/views/performance/newTraceDetails/traceDrawer/details/durationComparison.tsx

```diff
@@ -0,0 +1,68 @@
+import styled from '@emotion/styled';
+
+import {Row} from 'sentry/components/events/interfaces/spans/newTraceDetailsSpanDetails';
+import {t} from 'sentry/locale';
+import {space} from 'sentry/styles/space';
+import {getDuration} from 'sentry/utils/formatters';
+
+type Props = {
+  duration: number;
+  title: string;
+  avgDuration?: number;
+  toolTipText?: string;
+};
+
+export const COLORS = {
+  faster: {
+    light: 'green100',
+    normal: 'green300',
+  },
+  slower: {
+    light: 'red100',
+    normal: 'red300',
+  },
+  equal: {
+    light: 'gray100',
+    normal: 'gray300',
+  },
+};
+
+function DurationComparison(props: Props) {
+  if (!props.avgDuration) {
```

Comment:
`typeof props.avgDuration !== number` and you might want to check for Number.isNaN if you want to be safe :)